### DISCUSSION TITLE
Change `fireEvent` to `user` in the `editable` component test

### DISCUSSION THREAD

**Discussion by ghost** [chakra-ui/chakra-ui#8546] [General] [answered]
### Description

Currently, `fireEvent` is used to test the `editable` component.
We think this should be changed to `user`.

### Link to Reproduction

https://github.com/chakra-ui/chakra-ui/blob/main/packages/react/__tests__/editable.test.tsx

### Steps to reproduce

2. Click on '...'
3. Scroll down to '...'

### Chakra UI Version

2.8.2

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

**Comment by ghost** [comment]
I would like to challenge this issue.

**Comment by segunadebayo** [comment, marked as answer]
That test file is v2. We are not changing v2 tests.

v3 Editable tests live under the current package and already use user-event where it matters.
